hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/utils/TextFileBasedIdentityHandler.java

```diff
@@ -186,6 +188,14 @@ private static void loadMap(HashMap<String, String> cache, String fileLocation,
       LOG.debug("Loaded map stats - File: {}, Loaded: {}, Error: {} ", fileLocation, cache.size(), errorRecord);
     } catch (ArrayIndexOutOfBoundsException e) {
       LOG.error("Error while parsing mapping file", e);
+    } finally {
+      try {
```

Comment:
if it is an instance of closeable, we could use IOUtils.cleanupWithLogger() here

Reply:
done, thank you!